### DISCUSSION TITLE
Compatibility with Qt

### DISCUSSION THREAD

**Discussion by tortoisedoc** [Safetorun/safe_to_run#97] [Ideas]
Hello!
What would be the best way to integrate this libary for a qt app?

**Comment by dllewellyn** [comment]
That's a good question - I don't have much experience with Qt apps - are you able to call java / kotlin code from a qt app?

**Reply by tortoisedoc** [reply]
Qt provides their own main activity. I guess it would need some work there first.

**Comment by dllewellyn** [comment]
Going to create an issue for this - likely be down the list of priorities though unless someone with qt would be keen to pick it up
